SignalMessaging/utils/ThreadUtil.m

```diff
@@ -93,44 +93,45 @@ + (TSOutgoingMessage *)enqueueMessageWithAttachment:(SignalAttachment *)attachme
                                            inThread:(TSThread *)thread
                                    quotedReplyModel:(nullable OWSQuotedReplyModel *)quotedReplyModel
 {
-    return [self enqueueMessageWithAttachment:attachment
-                                     inThread:thread
-                             quotedReplyModel:quotedReplyModel
-                                 ignoreErrors:NO];
+    return [self enqueueMessageWithAttachment:attachment inThread:thread quotedReplyModel:quotedReplyModel];
```

Comment:
This looks like infinite recursion. 

I was expecting something like: `enqueueMessageWithAttachment**s**:@[ attachment ] ...`

Reply:
Done.